pkg/apis/experimental/v1/types.go

```diff
@@ -541,3 +541,64 @@ type IngressBackend struct {
 	// Specifies the protocol of the referenced service.
 	Protocol v1.Protocol `json:"protocol,omitempty"`
 }
+
+// DerivedNodeMetricsList contains derived metrics for a single node. It holds
+// derived metrics in windows about the node itself and all of its system
+// containers.
+type DerivedNodeMetrics struct {
+	unversioned.TypeMeta `json:",inline"`
+	// Standard list metadata.
+	v1.ObjectMeta `json:"metadata,omitempty"`
+
+	// NodeMetrics contains derived metrics about the node.
+	NodeMetrics MetricsWindows `json:"nodeMetrics"`
+	// SystemContainers contains derived container metrics for all system
+	// containers in the node.
+	SystemContainers []DerivedContainerMetrics `json:"systemContainers"`
+}
+
+// DerivedNodeMetricsList contains derived metrics for a single container.
+type DerivedContainerMetrics struct {
+	// ContainerName is the name of the container.
```

Comment:
@bgrant0607: You are correct. I was thinking of PodMetrics which is not relevant right now. 
For this PR, the name of the container refers to the raw cgroups name that is assigned by the kubelet. System containers are artifacts that are created by the kubelet, which is not known to the rest of the system. 
This field will eventually be overloaded to contain cgroup names and kubernetes pod container names. 
For now, we can document this field to be the `name of the raw cgroups that are managed by the kubelet on the node`.